reana_commons/openapi_specifications/reana_server.json

```diff
@@ -961,6 +961,69 @@
       }
     },
     "/api/workflows/{workflow_id_or_name}/workspace/{file_name}": {
+      "delete": {
+        "description": "This resource is expecting a workflow UUID and a filename existing inside the workspace to be deleted.",
+        "operationId": "delete_file",
```

Comment:
Would be nice to use "remove" for consistency with client, but we can tackle these during https://github.com/reanahub/reana-client/issues/213